### ISSUE TITLE
Option for changing listening port

### ISSUE THREAD

**Issue by klaymen00** [v0idp/Mellow#32]
Port 5060 is a common SIP port, and is blocked by some providers. Would be nice to have the option to change the listening port to get around that.

**Comment by christianfds** [comment]
If you run it through docker you can simply change the port binding in the **-p** option, so if you want to bind the container port 5060 to the host port 51234, you should do it like that:
```
docker run -d --restart=always --name mellow \
   -v /opt/appdata/mellow/:/usr/src/app/data/ \
   -p 51234:5060 \
   voidp/mellow
```

**Comment by dear-clouds** [comment]
Or you could just edit this line in WebServer.js

`this.app.listen(5060, this.onReady());`

**Comment by v0idp** [comment]
Closed due inactivity
